events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -0,0 +1,185 @@
+                <div id="typeDropdown" class="dropdown-items">
+                    <li>Any Type</li>
+                    <li>Multiple Choice</li>
+                    <li>True or False</li>
+                </div>
+            </ul>
+            <br>
+            <input type="button" id="selectedDifficulty" onclick="toggleMenu('difficultyDropdown')" class="gray" value="Any Difficulty   &#9660">
+            <ul class="dropdown-difficulty">
+                <div id="difficultyDropdown" class="dropdown-items">
+                    <li>Any Difficulty</li>
+                    <li>Easy</li>
+                    <li>Medium</li>
+                    <li>Hard</li>
+                </div>
+            </ul>
+            <br>
+            <input type="button" id="selectedCategory" onclick="toggleMenu('categoryDropdown')" class="gray" value="Any Category   &#9660">
+            <ul class="dropdown-category">
+                <div id="categoryDropdown" class="dropdown-items">
+                    <li value="9">General Knowledge</li>
+                    <li value="10">Books</li>
+                    <li value="11">Film</li>
+                    <li value="12">Music</li>
+                    <li value="13">Musicals and Theatres</li>
+                    <li value="14">Television</li>
+                    <li value="15">Video Games</li>
+                    <li value="16">Board Games</li>
+                    <li value="29">Comics</li>
+                    <li value="31">Japanese Anime and Manga</li>
+                    <li value="32">Cartoon and Animations</li>
+                    <li value="18">Computers</li>
+                    <li value="19">Mathematics</li>
+                    <li value="30">Gadgets</li>
+                    <li value="25">Art</li>
+                    <li value="26">Celebrities</li>
+                    <li value="27">Animals</li>
+                    <li value="28">Vehicles</li>
+                    <li value="20">Mythology</li>
+                    <li value="21">Sports</li>
+                    <li value="23">History</li>
+                    <li value="22">Geography</li>
+                    <li value="24">Politics</li>
+                </div>
+            </ul>
+            <br>
+        </div>
+        <p>
+        <input type="button" class="gray" id="newQuestion" value="New Question">
+        <p>
+        <input type="button" class="gray" id="showQuestion" value="Show This Question">
+        <input type="button" class="gray" id="showAnswers" value="Show The Answers">
+        <div id="question">
+            Your question will show here for preview. Make sure the question and answers do not have special 
+            characters and answers should not be longer than about 2/3 of this line.
+        </div>
+        <p>
+        <div id="answer">
+            Answer options will appear here.
+        </div>
+        <br>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
Don't change it in this code here, but please be mindful of jQuery usage. It doesn't _seem_ to be adding much value to this script.

Reply:
I haven't done anything with this html, Rebecca wrote it, but I will keep it in mind for the future!